sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogOptions.java

```diff
@@ -279,4 +280,28 @@ public HttpLogOptions setResponseLogger(HttpResponseLogger responseLogger) {
         this.responseLogger = responseLogger;
         return this;
     }
+
+    /**
+     * Sets the flag that controls if header names which value is redacted should be logged.
+     * <p>
+     * Applies only if logging request and response headers is enabled. See {@link HttpLogOptions#setLogLevel(HttpLogDetailLevel)} for details.
+     * Defaults to `false` - redacted header names are logged.
+     *
+     * @param disableRedactedHeaderLogging If true, redacted headers are not logged.
+     * Otherwise, headers with redacted values are logged with `REDACTED` value.
+     * @return The updated HttpLogOptions object.
+     */
+    public HttpLogOptions disableRedactedHeaderLogging(boolean disableRedactedHeaderLogging) {
+        this.disableRedactedHeaderLogging = disableRedactedHeaderLogging;
+        return this;
+    }
+
+    /**
+     * Gets the flag that controls if header names with redacted values should be logged.
+     *
+     * @return true if header names with redacted values are not be logged.
+     */
+    public boolean isRedactedHeaderLoggingEnabled() {
```

Comment:
nit: should we have symmetric getter for the setter method and return `isRedactedHeaderLoggingDisabled`?

Reply:
then every consumer would need to do `!isRedactedHeaderLoggingDisabled()` I can change the setter to `enableRedactedHeaderLogging` (with default true)